packages/core-js/internals/object-create.js

```diff
@@ -10,11 +10,34 @@ var IE_PROTO = sharedKey('IE_PROTO');
 var PROTOTYPE = 'prototype';
 var EmptyConstructor = function () { /* empty */ };
 
+var NullProtoObject = function (ProtoObject) {
+  var length = enumBugKeys.length;
+  while (length--) delete ProtoObject[PROTOTYPE][enumBugKeys[length]];
+  return ProtoObject();
+};
+
+// Create object with fake `null` prototype: use ActiveX Object with cleared prototype
+var NullProtoObjectViaActiveX = function () {
+  // Check for document.domain and active x support
+  // No need to use active x approach when document.domain is not set
+  // see https://github.com/es-shims/es5-shim/issues/150
+  // variation of https://github.com/kitcambridge/es5-shim/commit/4f738ac066346
+  try {
+    var activeXDocument = document.domain && new ActiveXObject('htmlfile');
```

Comment:
If it will throw an error, `ActiveXObject` will be created on each `Object.create(null)` even if required `IFrame`.

Reply:
And what is the suggestion? Create only one ActiveXObject for used?

Reply:
It seems there is the same question via IFrame

Reply:
check the latest commit

Reply:
Sure, it should be created only once.